server/src/models/Review.js

```diff
@@ -0,0 +1,35 @@
+const Model = require("./Model.js")
+
+class Review extends Model {
+  static get tableName() {
+    return "reviews"
+  }
+  
+  static get jsonSchema() {
+    return {
+      type: "object",
+      required: ["reviewBody", "score", "showId"],
+      properties: {
+        reviewBody: { type: "string", minLength: 10 },
+        score: { type: "integer" }
```

Comment:
score should between 1 and 5

Reply:
I did a google search, and we can use { type: "integer", minimum: 1, maximum: 5 } for this

Reply:
Okay ill add that in